src/dropdown/dropdown.js

```diff
@@ -85,6 +93,14 @@ angular.module('ui.bootstrap.dropdown', [])
     return self.toggleElement;
   };
 
+  scope.getAutoClose = function() {
+    return autoClose;
```

Comment:
Can just return `$attrs.autoClose` here. This will allow support for dynamic autoClose options. Though unlikely to be useful, we can save a variable and add support for that.